Swiftfin tvOS/Extensions/ButtonStyles/BorderlessFocus.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import SwiftUI
+
+struct BorderlessFocus: ButtonStyle {
```

Comment:
We should actually try to "avoid" making `ButtonStyle`s for tvOS. I would like us to stick to the default focus and scaling that the system provides unless we absolutely want a different design.

Making buttons is tricky in tvOS and actually quite annoying back when I was first working on the platform.

Reply:
No worries! That's kind of what I was wondering because this was a crazy amount of effort to get something that wasn't crazy looking. Doing this for every view would be... unpleasant... 

For now, do you want me to rollback this PR to just be the episodeCard spacing?